hiclass/LocalClassifierPerParentNode.py

```diff
@@ -223,3 +224,15 @@ def _fit_digraph(self, local_mode: bool = False, use_joblib: bool = False):
         self.logger_.info("Fitting local classifiers")
         nodes = self._get_parents()
         self._fit_node_classifier(nodes, local_mode, use_joblib)
+
+    def _get_predict_proba(model, X):
+        classifiers = [
+            model.hierarchy_.nodes[node]["classifier"]
+            for node in model.hierarchy_.nodes
+            if "classifier" in model.hierarchy_.nodes[node]
+        ]
+
+        # This will give list of target labels in the same order as predict_proba probabilities
+        # target_labels = [clf.classes_ for clf in classifiers]
+        predict_proba_per_classifier = [clf.predict_proba(X) for clf in classifiers]
```

Comment:
Thanks for implementing our suggestion! Would you mind changing this array to a dictionary with the node name as a key? Otherwise we don't know which probability belongs to which node.